packages/esm-patient-medications-app/src/order-basket/order-basket-search-results.scss

```diff
@@ -1,5 +1,64 @@
 @import "../root";
 
+.desktopSearchResultTile {
+  margin-top: 0.3125rem;
+  border-left: 4px solid $brand-teal-01;
+
+  &:not(:last-of-type) {
+    margin-bottom: 0.5rem;
+  }
+}
+
+.tabletSearchResultTile {
+  margin-top: 0.3125rem;
+  border-left: 2px solid $brand-teal-01;
+
+  &:not(:last-of-type) {
+    margin-bottom: 1rem;
+  }
+}
+
+.divider {
+  border: 0;
+  border-top: 1px solid $color-gray-30;
+  width: 66.66666667%;
```

Comment:
If you want to be exact just have 66.666666666666666666666666666666666666666666666666666666666666666666666666666666666666666666666666666666666666666666666666666666666666666666666666666666666666666666666666666666666666666666666666666666666666666666666...%

I guess overall 66.67 or so would be exact enough. In general though I am not sure we need to be that exact. A 66% would be sufficient, too.